helmfile.d/config/kusama/registrar-challenger-values.yaml.gotmpl

```diff
@@ -0,0 +1 @@
+ 
```

Comment:
Empty file?

Reply:
Same with `helmfile.d/config/polkadot/registrar-challenger-values.yaml.gotmpl`

Reply:
As you can see the helmfile.d/config/ folder is then splitted in kusama, polkadot, registrar-test.
In these folder there will be the specific config of each scenario.... for now kusama and polkadot are empty because our application is connected only with the registrar-test network ( the one pantelis and federico set up few days ago )

You can find a similar example here: https://github.com/w3f/polkadot-registrar-watcher/tree/master/helmfile.d/config